samples/snippets/src/main/java/com/example/spanner/CreateInstanceExample.java

```diff
@@ -65,6 +65,8 @@ static void createInstance(String projectId, String instanceId) {
           instanceInfo.getId(), e.getMessage());
     } catch (InterruptedException e) {
       System.out.println("Error: Waiting for createInstance operation to finish was interrupted");
+    } finally {
+      spanner.close();
```

Comment:
I presume your not using try-with-resources syntax because users are likely to keep spanner open across methods.  If so, LGTM, if not, consider using `try (spanner = ...) { ... }` syntax.

Reply:
Correct, a `Spanner` instance should preferably be kept open during the entire lifetime of the application, so we don't want to 'encourage' quickly opening and closing a `Spanner` instance.